src/MissionEditor/MissionItemStatus.qml

```diff
@@ -110,5 +146,60 @@ Rectangle {
                 }
             }
         }
+
+        Grid {
+            id:                 valueMissionGrid
+            columns:            2
+            columnSpacing:      _margins
+            anchors.verticalCenter: parent.verticalCenter
+
+            QGCLabel { text: qsTr("Total mission") }
```

Comment:
Could you add a (WIP) at the end of this to give folks a hint about fake values?